### DISCUSSION TITLE
Aktuelle Vorab-Links zu FRITZ!OS 8.02

### DISCUSSION THREAD

**Discussion by LizenzFass78851** [Freetz-NG/freetz-ng#1082] [Allgemein] [answered]
## Diese Liste enthält die Links für die folgende Modelle für das FRITZ!OS 8.02 Update

> [!NOTE]
> - Großes Dankeschön an [Grisu_](https://www.ip-phone-forum.de/members/grisu_.444642/) und alle beteiligten von [ip-phone-forum.de](https://www.ip-phone-forum.de) 
> - Was ein Vorab-Link ist, wurde in folgenden [ip-phone-forum.de thread](https://www.ip-phone-forum.de/threads/details-zu-den-8-02-firmwares-und-deren-downloadpfaden.320703/#post-2584312) erklärt.
> 
> - **Eine Liste an Geräten für die das FRITZ!OS 8.02 (oder höher) offiziell veröffentlicht wurden, ist unter folgenden [Link](https://github.com/Freetz-NG/freetz-ng/blob/master/docs/ftp/README.md) zu finden**

> [!TIP]
> - [Deskmodder](https://www.deskmodder.de/blog/) hatte bereits auch darüber [berichtet](https://www.deskmodder.de/blog/2024/12/23/naechstes-update-avm-veroeffentlicht-fritzos-8-02-fuer-diverse-modelle/)

> [!IMPORTANT]
> - Stand: 12.01.2025 14:30 Uhr

> [!WARNING]
> - Keine Garantie auf Funktionsfähigkeit und Aktualität der Links und deren Auflistung

#### Liste aller Modelle mit Vorab-Links
| Modell | Download | Changelog |
|:------------------ |:-------------- |:----------------- |
| 5530 Fiber | [FRITZ.Box_5530_Fiber-08.02.image](https://download.avm.de/firmware/5530/QkM7Q8lH0X/FRITZ.Box_5530_Fiber-08.02.image) | [info_de.txt](https://download.avm.de/firmware/5530/QkM7Q8lH0X/info_de.txt) |
| 5590 Fiber | [FRITZ.Box_5590_Fiber-08.02.image](https://download.avm.de/firmware/5590/SpvPusv3gx/FRITZ.Box_5590_Fiber-08.02.image) | [info_de.txt](https://download.avm.de/firmware/5590/SpvPusv3gx/info_de.txt) |
| 5690 Pro | [FRITZ.Box_5690_Pro-08.02.image](https://download.avm.de/firmware/5690pro/bHuoagBKtg/FRITZ.Box_5690_Pro-08.02.image) | [info_de.txt](https://download.avm.de/firmware/5690pro/bHuoagBKtg/info_de.txt) |
| 6591 Cable | [FRITZ.Box_6591_Cable-08.02.image](https://download.avm.de/firmware/6591/qstyFCmacr/FRITZ.Box_6591_Cable-08.02.image) | [info_de.txt](https://download.avm.de/firmware/6591/qstyFCmacr/info_de.txt) [info_en.txt](https://download.avm.de/firmware/6591/qstyFCmacr/info_en.txt) |
| 6850 LTE | [FRITZ.Box_6850_LTE-08.02.image](https://download.avm.de/firmware/jz137393/6850LTE/FRITZ.Box_6850_LTE-08.02.image) | [info_de.txt](https://download.avm.de/firmware/jz137393/6850LTE/info_de.txt) [info_en.txt](https://download.avm.de/firmware/jz137393/6850LTE/info_en.txt) |
| 6850 5G | [FRITZ.Box_6850_5G-08.02.image](https://download.avm.de/firmware/jz137393/68505G/FRITZ.Box_6850_5G-08.02.image) | [info_de.txt](https://download.avm.de/firmware/jz137393/68505G/info_de.txt) [info_en.txt](https://download.avm.de/firmware/jz137393/68505G/info_en.txt) |
| 7530 | [FRITZ.Box_7530-08.02.image](https://download.avm.de/firmware/7530/goqXLVBKhg/FRITZ.Box_7530-08.02.image) | [info_de.txt](https://download.avm.de/firmware/7530/goqXLVBKhg//info_de.txt) |
| 7530 AX | [FRITZ.Box_7530_AX-08.02.image](https://download.avm.de/firmware/7530AX/pR9lGwkdlX/FRITZ.Box_7530_AX-08.02.image) | [info_de.txt](https://download.avm.de/firmware/7530AX/pR9lGwkdlX/info_de.txt) [info_en.txt](https://download.avm.de/firmware/7530AX/pR9lGwkdlX/info_en.txt) |
| 7583 | [FRITZ.Box_7583-08.02.image](https://download.avm.de/firmware/7583/OtfZWov855/FRITZ.Box_7583-08.02.image) | [info_de.txt](https://download.avm.de/firmware/7583/OtfZWov855/info_de.txt) [info_en.txt](https://download.avm.de/firmware/7583/OtfZWov855/info_en.txt) |
| 7583 VDSL | [FRITZ.Box_7583_VDSL-08.02.image](https://download.avm.de/firmware/7583VDSL/AtnkrJW867/FRITZ.Box_7583_VDSL-08.02.image) | [info_de.txt](https://download.avm.de/firmware/7583VDSL/AtnkrJW867//info_de.txt) [info_en.txt](https://download.avm.de/firmware/7583VDSL/AtnkrJW867//info_en.txt) |
| 7590 | [FRITZ.Box_7590-08.02.image](https://download.avm.de/firmware/7590/td53w9F2GL/FRITZ.Box_7590-08.02.image) | [info_de.txt](https://download.avm.de/firmware/7590/td53w9F2GL/info_de.txt) |
| 7590 AX | [FRITZ.Box_7590_AX-08.02.image](https://download.avm.de/firmware/7590AX/Xa0NWpjdTF/FRITZ.Box_7590_AX-08.02.image) | [info_de.txt](https://download.avm.de/firmware/7590AX/Xa0NWpjdTF/info_de.txt) |
| 7690 | [FRITZ.Box_7690-08.02.image](https://download.avm.de/firmware/7690/nsF7Sc2oLB/FRITZ.Box_7690-08.02.image) | [info_de.txt](https://download.avm.de/firmware/7690/nsF7Sc2oLB/info_de.txt) |

<details>
  <summary>Liste aller Modelle ohne Vorab-Links</summary>

| Modell |
|:------------------ |
| 4040       |
| 4050       |
| 4060       |
| 6660 Cable |
| 6670 Cable |
| 6690 Cable |
| 7510       |
| 7520       |
| 7520 B     |
| 7682       |

<details>
  <summary>Unveröffentlichte Modelle</summary>

| Modell |
|:------------------ |
| 7530 AX B  |
| 7539  |

</details>

</details>

**Comment by fda77** [comment]
Die 8.00 litt unter "spontanen Reboots",
mit der 8.01 ging Pushmail nur teilweise und Anrufe nach Österreich machen Probleme.
Die 8.02 soll laut Kommentaren bei Deskmodder Problem mit den LEDs haben.
Vielleicht kommt dann zu Silvester gleich 8.04 auf die offiziellen Server.
Und nächstes Jahr führt AVM eine Qualitätsmanagement ein ;-)

Wie findest man eigentlich diese URLs? Ich hab das Juis-Script duchrlaufen lassen, aber das fand nichts: https://github.com/Freetz-NG/freetz-ng/actions/runs/12482268282/job/34836099183

**Reply by LizenzFass78851** [reply]
Habe ich auch schon versucht aber laut [Grisu_'s Beitrag](https://www.ip-phone-forum.de/threads/fritz-box-7530-fritz-os-8-0x-vom-10-10-2024.319989/page-2#post-2583658) muss man:
- als Land die `043`, 
- als oem die `avme` und 
- als UpdateConfig die `3` 

definieren aber selbst damit hat es bei mir bei besagten Modellen nicht die 8.02 gefunden.
Links stammen aus [ip-phone-forum.de](https://www.ip-phone-forum.de/)

**Reply by evilbaschdi** [reply]
das mit den reboots ist mir der 8.0.2 auf meiner 7590 nicht wirklich besser. hat da jemand schon infos zu nem workaround? morgen geht das ticket bei avm wieder auf.

**Comment by fda77** [comment]
Mit "Country=043" bekomme ich nicht mal die 8.00er! 
Es gibt aber noch einige andere Einstellungen die AVM da so mitbekommt, wie "Provider" oder "ProviderName"

Das "juis_check" in freetz ist übrigens nicht das von peterpawn, aus kompatibilitätsgründen hat es aber die gleiche Parameterübergabe + zusätzliche Parameter
Evtl gehts nur mit dem anderen.

8.02 finde ich jedenfalls keine:
```
for x in $(seq 200 290); do echo -n "$x " &&  env - tools/juis_check HW=$x Version=08.00 Country=049 OEM=avme UpdateConfig=3 -s; done
200 Found newer version: 128.07.15
201 Found newer version: 129.07.15
202 No newer version found.
203 No newer version found.
204 No newer version found.
205 Found newer version: 133.07.04
206 Found newer version: 134.07.32
207 No newer version found.
208 No newer version found.
209 No newer version found.
210 No newer version found.
211 No newer version found.
212 Found newer version: 140.07.12
213 No newer version found.
214 Found newer version: 142.07.59
215 Found newer version: 143.07.16
216 Found newer version: 144.07.15
217 No newer version found.
218 Found newer version: 146.07.12
219 Found newer version: 147.07.04
220 No newer version found.
221 Found newer version: 149.07.12
222 Found newer version: 150.07.16
223 Found newer version: 151.07.31
224 Found newer version: 152.07.18
225 No newer version found.
226 Found newer version: 154.07.13
227 Found newer version: 227.07.59
228 Found newer version: 156.07.18
229 Found newer version: 157.07.58
230 No newer version found.
231 No newer version found.
232 No newer version found.
233 No newer version found.
234 Found newer version: 162.07.57
235 No newer version found.
236 Found newer version: 164.08.00
237 No newer version found.
238 No newer version found.
239 Found newer version: 167.08.00
240 Found newer version: 168.07.58
241 Found newer version: 169.07.58
242 No newer version found.
243 Found newer version: 171.07.31
244 Found newer version: 172.07.58
245 No newer version found.
246 Found newer version: 174.07.58
247 No newer version found.
248 No newer version found.
249 Found newer version: 249.07.58
250 No newer version found.
251 No newer version found.
252 No newer version found.
253 Found newer version: 253.07.58
254 No newer version found.
255 No newer version found.
256 Found newer version: 256.08.00
257 Found newer version: 257.08.00
258 Found newer version: 258.08.00
259 Found newer version: 259.08.00
260 Found newer version: 260.08.00
261 Found newer version: 261.07.59
262 Found newer version: 262.08.00
263 Found newer version: 263.07.58
264 No newer version found.
265 No newer version found.
266 No newer version found.
267 No newer version found.
268 Found newer version: 268.07.58
269 No newer version found.
270 Found newer version: 270.07.58
271 Found newer version: 271.08.00
272 Found newer version: 272.08.00 Release
273 No newer version found.
274 No newer version found.
275 Found newer version: 275.07.58
276 No newer version found.
277 No newer version found.
278 No newer version found.
279 No newer version found.
280 No newer version found.
281 No newer version found.
282 No newer version found.
283 No newer version found.
284 No newer version found.
285 No newer version found.
286 No newer version found.
287 No newer version found.
288 No newer version found.
289 No newer version found.
290 No newer version found.
```

**Reply by LizenzFass78851** [reply]
Mittlerweile habe ich mit folgenden Befehl:
```bash
for x in $(seq 222 333); do  [ "$x" -lt 248 ] 2>/dev/null && m="$(( $x - 72 ))" || m=$x; m="$m.08.00" 
  env - ./tools/juis_check \
    HW=$x \
    Buildtype=1 \
    OEM=avme \
    Country=043 \
    UpdateConfig=3 \
    Version=$m  \
  -a; done
```

```
226=http://download.avm.de/firmware/7590/td53w9F2GL/FRITZ.Box_7590-08.02.image
236=http://download.avm.de/firmware/7530/goqXLVBKhg/FRITZ.Box_7530-08.02.image
260=http://download.avm.de/firmware/7583VDSL/AtnkrJW867/FRITZ.Box_7583_VDSL-08.02.image
```

nun bei 3 Modellen die 8.02 finden können aber bei weiteren Modellen wie die z.b. 7530 AX wird weiterhin nichts neueres gefunden.

-----

Eine `juis_boxinfo.xml` habe ich mir für das andere juis Tool zusammengebastelt:

```xml
<e:BoxInfo xmlns:e="http://juis.avm.de/updateinfo" xmlns:q="http://juis.avm.de/request">
  <q:Name>FRITZ!Box 7530 (UI)</q:Name>
  <q:HW>236</q:HW>
  <q:Major>164</q:Major>
  <q:Minor>8</q:Minor>
  <q:Patch>0</q:Patch>
  <q:Buildnumber>100000</q:Buildnumber>
  <q:Buildtype>1</q:Buildtype>
  <q:Serial>3CA62FXXXXXX</q:Serial>
  <q:OEM>avme</q:OEM>
  <q:Lang>de</q:Lang>
  <q:Country>043</q:Country>
  <q:Annex>A</q:Annex>
  <q:Flag>medium_lan</q:Flag>
  <q:Flag>mesh_repeater_no_trusted</q:Flag>
  <q:UpdateConfig>3</q:UpdateConfig>
  <q:Provider>a1</q:Provider>
  <q:ProviderName>A1 Telekom Austria</q:ProviderName>
</e:BoxInfo>
```

Damit habe ich auch beim anderen juis Tool die besagte 8.02 gefunden.
Gleiches an die Werte für eine 7530 AX angepasst, hat auch mit den anderen juis Tool nichts neueres gefunden.

**Comment by fda77** [comment]
```
FRITZ.Box_5690_Pro-08.02.image    5690-Pro_de-es-fr-it-nl-pl      279.08.02 rev117968 {ALL} [Smart24 P1 FCS]        <20.12.2024 16:48:12>
FRITZ.Box_7690-08.02.image        7690_de-es-fr-it-nl-pl          285.08.02 rev117971 {ALL} [Smart24 P1 FCS]        <20.12.2024 16:48:12>
FRITZ.Box_7590-08.02.image        7590_de-es-fr-it-nl-pl          154.08.02 rev117975 {ALL} [Smart24 P1 NL1]        <20.12.2024 16:46:16>
FRITZ.Box_6591_Cable-08.02.image  6591-Cable_de-es-fr-it-pl       161.08.02 rev117977 {GER} [Smart24 P1 NL1]        <20.12.2024 16:46:16>
FRITZ.Box_7530-08.02.image        7530_de-es-fr-it-nl-pl          164.08.02 rev117978 {ALL} [Smart24 P1 NL1]        <20.12.2024 16:46:16>
FRITZ.Box_7583-08.02.image        7583_de-es-fr-it-nl-pl          167.08.02 rev117979 {ALL} [Smart24 P1 NL1]        <20.12.2024 16:46:16>
FRITZ.Box_7530_AX-08.02.image     7530-AX_de-es-fr-it-nl-pl       256.08.02 rev117981 {ALL} [Smart24 P1 NL1]        <20.12.2024 16:46:16>
FRITZ.Box_5530_Fiber-08.02.image  5530-Fiber_de-es-fr-it-nl-pl    257.08.02 rev117982 {ALL} [Smart24 P1 NL1]        <20.12.2024 16:46:16>
FRITZ.Box_6850_5G-08.02.image     6850-5G_en-de-es-fr-it-nl-pl    258.08.02 rev117983 {ALL} [Smart24 P1 NL1]        <20.12.2024 16:46:16>
FRITZ.Box_7590_AX-08.02.image     7590-AX_de-es-fr-it-nl-pl       259.08.02 rev117984 {ALL} [Smart24 P1 NL1]        <20.12.2024 16:46:16>
FRITZ.Box_7583_VDSL-08.02.image   7583-VDSL_de-es-fr-it-nl-pl     260.08.02 rev117985 {ALL} [Smart24 P1 NL1]        <20.12.2024 16:46:16>
FRITZ.Box_6850_LTE-08.02.image    6850-LTE_en-de-es-fr-it-nl-pl   262.08.02 rev117986 {ALL} [Smart24 P1 NL1]        <20.12.2024 16:46:16>
FRITZ.Box_5590_Fiber-08.02.image  5590-Fiber_de-es-fr-it-nl-pl    272.08.02 rev117989 {ALL} [Smart24 P1 NL1]        <20.12.2024 16:46:16>
```

**Reply by LizenzFass78851** [reply]
~Es sind nun weitere Modelle gefunden worden:~
Ich habe weitere Modelle aufgelistet:
- 5690 Pro
- 6591 Cable

**edit**: Der Link für die 5690 Pro stammt ebenfalls aus [ip-phone-forum.de](https://www.ip-phone-forum.de) und für die 6591 Cable habe ich die Firmware mit den Befehl aus folgenden [Comment](https://github.com/Freetz-NG/freetz-ng/discussions/1082#discussioncomment-11663634) erhalten nur mit den Unterschied beim "oem" statt `avme`, `avm` anzugeben.

**Reply by fda77** [reply]
Wenn man sich die AVM-Revisionen anschaut dürften da noch einige kommen.
Scheint aber kein Sicherheitsproblem zu sein da die Veröffentlichung so zäh ist.

**Reply by LizenzFass78851** [reply]
Ich habe erneut weitere Modelle aufgelistet:
- 5530 Fiber
- 5590 Fiber

Die Links stammen ebenfalls aus [ip-phone-forum.de](https://www.ip-phone-forum.de)

**Comment by fda77** [comment]
Wenn man diese neuer Versionen ausprobieren möchte:
 - die .image nach dl/fw/ herunterladen
 - im menuconfig den Namen der Datei eintragen unter "Download Options" > "Override firmware options" > "Firmware source" 

Ich nutze das auf einer 7520-alien, läuft soweit unauffällig als modem

**Comment by fda77** [comment]
@LizenzFass78851 ich hab gar nicht mitbekommen dass du die tabelle erweitert hast

**Reply by LizenzFass78851** [reply]
Die verfügbaren Links sind weiterhin die gleichen, nur mit den unterscheid das ich in der Tabelle alle Fritzboxen aufliste, die OS 8.0 oder höher haben sowie die zwei 40er Modelle aus der Laborreihe.

**Reply by fda77** [reply]
Das meine ich ja :)
Sind das alle die überhaupt noch neue Funktionen (also FOS8) bekommen? Ich hab keinen Überblick mehr
EDIT: "7530 AX B" fehlt noch, die hat schon inhaus

**Reply by LizenzFass78851** [reply]
Warscheinlich.
Ich habe unter https://github.com/Freetz-NG/freetz-ng/blob/master/docs/ftp/README.md geschaut und von dort die Modelle aufgelistet die OS 8.0 oder höher haben.

7530 AX B, steht nun auf der [Liste](https://github.com/Freetz-NG/freetz-ng/discussions/1082#discussion-7739340).

**Comment by fda77** [comment]
Auf meiner 7520 die als 7530-alien zuerst 8.0, dann 8.0-plus und nun 8.02 hat, habe ich festgestellt dass sich mal wieder selbst irgend welche Rechte von Geisterhand erteilt wurden:

![image](https://github.com/user-attachments/assets/43caab6d-3228-4161-bbb1-aad806a8e0c9)

Da ich dies Gerät als "modem" nutze hat es zum Glück kein Zugriff auf mein lan

**Comment by fda77** [comment]
Scheinbar zieht AVM das gleich Spiel wie immer ab und veröffentlicht alle paar Tage eine weite Firmware die schon seit 1 Monate compiliert und bekannt sind -.-

@LizenzFass78851: Falls du gerne Listen machst...
Ein Überblick über die AVM-Sourcen für die kernelmodule wäre noch gut. Also für welche Geräte die Sourcen in Freetz noch geupdatet werden müssen, weil zb ubekannte Symbole (warnung) beim make auftreten. Ab Fos 8 reicht, Sprünge von 8.00 auf .01 oder .02 sind idr nicht nötig da mit Freetz nur Module und nicht mehr der ganze Kernel ersetzt werden kann.
Das trifft zb auf die 4050+4060 zu, dort werden noch alte Sourcen 7.5 genutzt

**Reply by LizenzFass78851** [reply]
Ich habe eine entsprechende Liste im Comment hinzugefügt.

<details>
  <summary>Liste der in Freetz aktuellen Sources für besagte Modelle</summary>

| Modell | Source für ≥08.00 vorhanden | 
|:------------------ |:------------------ |
| 5530 | ✔ | 
| 5590 | ✔ | 
| 5690 | ✔ | 
| 6591 | ✔ | 
| 6850 | ✔ | 
| 7530_W5 | ✔ | 
| 7530_W6 | ❌ | 
| 7539 | ✔ |
| 7583_V1 | ❌ | 
| 7583_V2 | ❌ | 
| 7590_W5 | ❌ | 
| 7590_W6 | ❌ | 
| 7690 | ✔ | 
| 4040 | ✔ | 
| 4050 | ❌ | 
| 4060 | ❌ | 
| 6660 | ❌ | 
| 6670 | ❌ | 
| 6690 | ❌ | 
| 7510 | ✔ | 
| 7520_V1 | ✔ | 
| 7520_V2 | ✔ | 
| 7530_W6_V2 | ✔ | 
| 7682 | ❌ | 

Quelle: https://github.com/Freetz-NG/freetz-ng/blob/master/config/avm/source.in

</details>

**Reply by fda77** [reply]
Hm, da gibt es ein kleines Missverständnis. Welche es gibt sind schon klar. Vielmehr weisst ich nicht welche man wirklich noch updaten muss / für die es noch keine passenden Sourcen gibt! Das sind nämlich nur die die  Warnung "unbekannte Symbole" beim COMPILIEREN zeigen!
Und genau dieses compilieren von allen Kernel (man muss min 1 modul ausgewählt haben) braucht recht viel Zeit!

**Reply by fda77** [reply]
Am Beispiel https://github.com/Freetz-NG/freetz-ng/commit/a85f83f420da27706953b79910004501a7415bda  :
für 4050 ist bekannt dass sie fehlen und dass 7530ax kaputt sind. 7682 src sind neu und ich weiss nicht ob die aktuell genutzten von einer anderen box passen

**Reply by LizenzFass78851** [reply]
Ich habe testweiße ein GitHub Workflow mit Matrix erstellt um das Testen des Fehlerbilds bez. "unbekannte Symbole" zu beschleunigen.

Heraus kommt folgende Logs bei denen der Fehler `WARNING: Unresolved symbols detected` auftaucht.

- [7690.txt](https://github.com/user-attachments/files/18428407/7690.txt)
- [7590_W5.txt](https://github.com/user-attachments/files/18428410/7590_W5.txt)
- [7520_V1.txt](https://github.com/user-attachments/files/18428412/7520_V1.txt)
- [7510.txt](https://github.com/user-attachments/files/18428413/7510.txt)
- [4060.txt](https://github.com/user-attachments/files/18428414/4060.txt)
- [4040.txt](https://github.com/user-attachments/files/18428415/4040.txt)
- [7590_W6.txt](https://github.com/user-attachments/files/18428417/7590_W6.txt)
- [7583_V2.txt](https://github.com/user-attachments/files/18428418/7583_V2.txt)
- [7583_V1.txt](https://github.com/user-attachments/files/18428419/7583_V1.txt)
- [7530_W5.txt](https://github.com/user-attachments/files/18428420/7530_W5.txt)
- [5690_PRO.txt](https://github.com/user-attachments/files/18428421/5690_PRO.txt)
- [5590.txt](https://github.com/user-attachments/files/18428431/5590.txt)
- [6591.txt](https://github.com/user-attachments/files/18428529/6591.txt) (möglich aber wegen #1071 nicht testbar)
- [6660.txt](https://github.com/user-attachments/files/18428526/6660.txt) (möglich aber wegen #1071 nicht testbar)
- [6690.txt](https://github.com/user-attachments/files/18428528/6690.txt) (möglich aber wegen #1071 nicht testbar)

Ich habe dazu fast alle Kernelmodule dabei ausgewählt.
Testzeitpunkt war mit folgenden Commit 34e6c85ee3e9cd6502458ebeacc69a3c72d572b5. (Neuere Commits wurden nicht mit einbezogen.)

**Reply by fda77** [reply]
Die "matrix" kannte ich noch gar nicht, prima Sache!! Hab nur leider direkt meinen Account für den Rest des Monats verbraucht -.-
Am Ende der "GITHUB_TOKEN" ist ein Secret das man auf github.com für das Repo setzen musse, das ist ein AuthToken von einem extra Maschinenaccount https://github.com/machine-ng weil des mit dem normalen Token nicht löschbar ist...
Bei GIthub kann man nämlich kein neues Cache anlegen wenn es das schon gibt ("überschreiben")

Zuerst teste ich nur mit einem kleinen modul, zb "ftdi" um zu schauen ob die Sourcen überhaupt zu Firmware passen.
Ausgeschlossen hab ich "corrupet" sourcen die gar nicht compilierbar sind und "tntested" Geräte - wenn es niemadn nutzt brauch man auch keine sourcen für kernelmodule. Soweit sieht das ganz gut aus

Nach dem UDF und ARC4 müsste man separat schauen. Mit jeder Kernelversion ändern sich die Namen etwas, es kommen neu Abhängigkeiten und andere fallen weg

Ohne Kernel recht flott: https://github.com/Freetz-NG/freetz-ng/actions/runs/12803822055

Falls du https://github.com/Freetz-NG/freetz-ng/actions/runs/12804270109/workflow verwenden möchtest, lösch FREETZ_DOWNLOAD_TOOLCHAIN (und FREETZ_REAL_DEVELOPER_ONLY__DLURL)

**Comment by LizenzFass78851** [comment, marked as answer]
AVM hat inzwischen für alle Modelle die FRITZ!OS Version 8.02 oder höher (bzw. eine entsprechende Laborverson für 8.03) veröffentlicht, für die es [Vorab-Links](https://github.com/Freetz-NG/freetz-ng/discussions/1082#discussion-7739340) zu 8.02 gab.

**Comment by fda77** [comment]
Es fehlen noch diese:
```
FRITZ.Box_5690_Pro-08.02.image          5690-Pro_de-es-fr-it-nl-pl              279.08.02 rev117968 {ALL} [Smart24 P1 FCS]             <20.12.2024 16:48:12>
FRITZ.Box_7583-08.02.image              7583_de-es-fr-it-nl-pl                  167.08.02 rev117979 {ALL} [Smart24 P1 NL1]             <20.12.2024 16:46:16>
FRITZ.Box_6850_5G-08.02.image           6850-5G_en-de-es-fr-it-nl-pl            258.08.02 rev117983 {ALL} [Smart24 P1 NL1]             <20.12.2024 16:46:16>
FRITZ.Box_7583_VDSL-08.02.image         7583-VDSL_de-es-fr-it-nl-pl             260.08.02 rev117985 {ALL} [Smart24 P1 NL1]             <20.12.2024 16:46:16>
FRITZ.Box_6850_LTE-08.02.image          6850-LTE_en-de-es-fr-it-nl-pl           262.08.02 rev117986 {ALL} [Smart24 P1 NL1]             <20.12.2024 16:46:16>
```
Im labor sind nur die betaversionen von 6850 4g+5g, 3 fehlen noch komplett. Die "inhaus" alphaversionen hab ich nicht im blick

**Reply by LizenzFass78851** [reply]
Die 5690 Pro hat heute die 8.03 erhalten.
Quelle: https://www.deskmodder.de/blog/2025/02/06/fritzbox-5690-pro-erhaelt-das-fritzos-8-03/

**Comment by fda77** [comment]
Achso! Die ftp action läuft noch... https://github.com/Freetz-NG/freetz-ng/actions/runs/13181810750/job/36794235969

**Reply by fda77** [reply]
Offenbar gibt es in der 8.03 nichts neues, das Update kann man sich sparen: https://download.avm.de/fritzbox/fritzbox-5690-pro/deutschland/fritz.os/info_de.txt

**Reply by fda77** [reply]
Und eigentlich wollte ich nach 8.02 eine freetz-tag ertsellen, aber das veröffentlichen bereits compilierter images bei avm zieht sich nun schon seit 2 Monaten hin!

**Reply by LizenzFass78851** [reply]
> Offenbar gibt es in der 8.03 nichts neues, das Update kann man sich sparen: https://download.avm.de/fritzbox/fritzbox-5690-pro/deutschland/fritz.os/info_de.txt

Einen zusätzlichen Eintrag in changelog im Vergleich zu 8.02 konnte ich ausfindig machen:

```diff
 ## Internet:
+- **Behoben** Bei Nutzung bestimmter Mobilfunksticks hat der Fallback auf Mobilfunk im Erweiterten Ausfallschutz nicht funktioniert
 - **Behoben** Manchmal wurden E-Mails über den Push Service oder MyFRITZ!Net mit leeren Anmelde-Informationen versendet
 
 ## Telefonie:
 - **Behoben** In Österreich (Country Code 43) war die Anwahl der Rufnummerngassen 12, 13 und 14 nur mit Blockwahl möglich
 - **Verbesserung** Stabilität von Telefonaten erhöht
```

**Comment by fda77** [comment]
Beim Builddatum ist AVM nicht so ganz ehrlich, eine neuere Revision wurde angeblich vor einer älteren gebaut

```
	FRITZ.Box_7682-08.03.image                   286.08.03 rev118255 {ALL} [Smart24 P1 FCS]             <15.01.2025 18:44:40>
	FRITZ.Box_7490-07.60.image                   113.07.60 rev118285 {ALL} [MOVE21NL2]                  <04.12.2024 17:55:07>
```

**Comment by fda77** [comment]
Jetzt sind alle 8.02 oder neuer veröffentlicht. 
Eine Besonderheit sind 6850 4G + 5G: Die gibt es offiziell nicht als .image sondern nur als reocvery.exe in der Labor 8.02...
